ouroboros-network/src/Ouroboros/Network/BlockFetch/ClientState.hs

```diff
@@ -488,12 +519,12 @@ newTMergeVar = TMergeVar <$> newEmptyTMVar
 -- | Merge the current value with the given one and store it, return the updated
 -- value.
```

Comment:
Stale comment still says `return the updated value`

Reply:
Ta.